recipes-connectivity/awsclient/awsclient/awsclientcheck.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+FILE="config/esec.config"
+
+if [ -f "${FILE}" ]; then
+    condition=$(jq .awsclient < $FILE | sed -r 's/["]+//g')
+    [ "$condition" = start ] && awsclient
```

Comment:
You could condense these two lines into one and insert that command directly into the service file. This way we do not need yet another shell script.

Reply:
is changed